tests/unit_tests/test_wave_02.py

```diff
@@ -2,30 +2,37 @@
 from swap_meet.vendor import Vendor
 from swap_meet.item import Item
 
-@pytest.mark.skip
+#@pytest.mark.skip
 def test_items_have_default_uuid_length_id():
     item = Item()
     assert isinstance(item.id, int)
     assert len(str(item.id)) >= 32
 
-@pytest.mark.skip
+#@pytest.mark.skip
 def test_item_instances_have_different_default_ids():
     item_a = Item()
     item_b = Item()
     assert item_a.id != item_b.id
 
-@pytest.mark.skip
+#@pytest.mark.skip
 def test_items_use_custom_id_if_passed():
     item = Item(id=12345)
     assert isinstance(item.id, int)
     assert item.id == 12345
 
-@pytest.mark.skip
+#@pytest.mark.skip
+def test_items_use_custom_id_not_int_not_none():
+    item = Item(id="hello, world")
+    assert isinstance(item.id, int)
+    assert len(str(item.id)) >= 32
```

Comment:
This is a great new test to add! Don't be afraid to add tests that cover cases you've added in!